src/zenml/integrations/kserve/steps/kserve_deployer.py

```diff
@@ -163,6 +169,37 @@ def torch_config_validate(cls, v: Optional[str]) -> Optional[str]:
         return v
 
 
+class CustomDeployParameters(BaseModel):
+    """Custom model deployer step extra parameters.
+
+    Attributes:
+        predict_function: Path to Python file containing predict function.
+    """
+
+    predict_function: str
+
+    @validator("predict_function")
+    def predict_function_validate(cls, predict_func_path: str) -> str:
+        """Validate predict function.
+
+        Args:
+            predict_func_path: predict function path
+
+        Returns:
+            predict function path
+
+        Raises:
+            ValueError: if predict function path is not valid
+        """
+        if not predict_func_path:
+            raise ValueError("Predict function path is required.")
+        try:
+            import_class_by_path(predict_func_path)
+        except AttributeError:
+            raise ValueError("Predict function can't be found.")
+        return predict_func_path
```

Comment:
Should we also add a check here to make sure the predict function is actually a function/something callable?

Reply:
I think that's a great idea! do you think this is enough @schustmi ?
```
        try:
            predict_function = import_class_by_path(predict_func_path)
        except AttributeError:
            raise ValueError("Predict function can't be found.")
        if not callable(predict_function):
            raise ValueError("Predict function must be callable.")
```

Reply:
Tiny nit: I would use a `TypeError` for the last one, as it is related to the value having the wrong type. Other than that looks good!